docs/_docs/2_4_configurations.md

```diff
@@ -55,7 +55,8 @@ Options useful for writing tables via `write.format.option(...)`
 | RECORDKEY_FIELD_OPT_KEY | hoodie.datasource.write.recordkey.field | NO | uuid | Record key field. Value to be used as the recordKey component of HoodieKey. Actual value will be obtained by invoking .toString() on the field value. Nested fields can be specified using the dot notation eg: a.b.c |
 | PARTITIONPATH_FIELD_OPT_KEY | hoodie.datasource.write.partitionpath.field | NO | partitionpath | Partition path field. Value to be used at the partitionPath component of HoodieKey. Actual value ontained by invoking .toString() |
 | HIVE_STYLE_PARTITIONING_OPT_KEY | hoodie.datasource.write.hive_style_partitioning | NO | false | If set true, the names of partition folders follow <partition_column_name>=<partition_value> format. |
-| KEYGENERATOR_CLASS_OPT_KEY | hoodie.datasource.write.keygenerator.class | NO | org.apache.hudi.keygen.SimpleKeyGenerator | Key generator class, that implements will extract the key out of incoming Row object. |
+| KEYGENERATOR_CLASS_OPT_KEY |  | NO | org.apache.hudi.keygen.SimpleKeyGenerator | Key generator class, that implements will extract the key out of incoming `Row` object. This config has higher precedence over keygen type, and it is used for user-defined KeyGenerator . |
```

Comment:
I thought we should remove "org.apache.hudi.keygen.SimpleKeyGenerator" from here and say "No default"

Reply:
> I thought we should remove "org.apache.hudi.keygen.SimpleKeyGenerator" from here and say "No default"

done